packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -90,7 +90,7 @@ This is a starting point for browser-based OpenID Connect flows such as the impl
 | [idp](/docs/reference/api/idps/) | Identity provider to use if there's no Okta Session.                                                                                                                                                                                                                                                                                                                  | Query       | String    | FALSE      |
 | login_hint                       | A username to prepopulate if prompting for authentication.                                                                                                                                                                                                                                                                                                                                           | Query       | String    | FALSE      |
 | max_age                          | Allowable elapsed time, in seconds, since the last time the end user was actively authenticated by Okta.                                                                                                                                                                                                                                                                                                                                           | Query       | String    | FALSE      |
-| nonce                            | A value that is returned in the ID token. It is used to mitigate replay attacks.                                                                                                                                                                                                                                                                                                                                                              | Query       | String    | TRUE       |
+| nonce                            | A value that is returned in the ID token. It is used to mitigate replay attacks. The value is required for Implicit and Hybrid flows, but optional for Auth Code flows. For more details, see the [OIDC Specs](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest).                                                                                                                                                                                                                                                                                                                                                             | Query       | String    | FALSE       |
```

Comment:
Looks good, but revise the link as per our styles, i.e.: See [link].

See Row 10 https://oktawiki.atlassian.net/wiki/spaces/DOC/pages/722929640/InfoDev+Style+Guide#3.-Best-practices-for-common-mistakes

Reply:
Thanks, Brian. Shaking off the influence of my last style guide lol